rasa_core/trackers.py

```diff
@@ -104,6 +104,8 @@ def __init__(self, sender_id, slots,
         self.latest_bot_utterance = None
         self._reset()
         self.active_form = {}
+        # Marking if story is original or generated
+        self.augmented = False
```

Comment:
do we need it for original tracker? I think it can be property of `TrackerWithCachedStates` only, maybe check for it in `Memoization.train`